assets/js/choice.js

```diff
@@ -0,0 +1,125 @@
+var CHoice;
+
+(function ($) {
```

Comment:
Why should we need `assets/js/choice.js` and `assets/js/modules/choice/choice.js`?

Reply:
`assets/modules/choice/choice.js` is the core source code, `assets/js/choice.js` the concatened source file before minification (of all JS files in choice, which, in this case, is indeed one file), which is then minified into `assets/js/choice.min.js`. We include the minified or full version based on the WP_DEBUG setting.